fusor/models.py

```diff
@@ -257,6 +257,8 @@ def validate_sequence(cls, v):
 
         return v
 
+    # TODO[pydantic]: The `Config` class inherits from another class, please create the `model_config` manually.
+    # Check https://docs.pydantic.dev/dev-v2/migration/#changes-to-config for more information.
```

Comment:
Resolve todos. [Example](https://github.com/GenomicMedLab/cool-seq-tool/blob/vrs-2.0-alpha/cool_seq_tool/schemas.py#L79-L91) from cool-seq-tool

Reply:
oop, just now seeing this. I worked through these yesterday and I think I got them all fixed?